CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Brush selection is now maintained when selecting a new variable.
 
+### Added
+- Image server now accepts startDT and endDT (ISO-8601 dates) to set the range of the hydrograph
```

Comment:
@JanellFry-usgs, I added a description of the work that you did to the CHANGELOG.md. I missed this in my review of your work.